core/src/main/kotlin/org/evomaster/core/EMConfig.kt

```diff
@@ -668,15 +673,15 @@ class EMConfig {
     enum class TestSuiteSplitType {
         NONE,
         CLUSTER,
-        SUMMARY_ONLY,
         CODE
     }
```

Comment:
what is the difference between CLUSTER and CODE? add comments here to clarify

Reply:
Will do.